### DISCUSSION TITLE
The Future of httpx: Adapting to Wappalyzer's Closed Source Status

### DISCUSSION THREAD

**Discussion by ceylanb** [projectdiscovery/httpx#1370] [General] [answered]
Given that httpx appears to rely on the former Wappalyzer database, which is no longer open source [[Source]](https://news.ycombinator.com/item?id=37236746), I'm curious about the future plans of httpx's maintainers and contributors. Do they anticipate the community surrounding httpx will address this challenge, and what are their thoughts on this matter?

**Comment by olearycrew** [comment]
@ceylanb we are still looking at all options - but there was a fork made that we've seen some community activity on: https://github.com/Lissy93/wapalyzer

**Comment by ehsandeep** [comment, marked as answer]
Hello @ceylanb, thank you for discussing this with us. We are aware of the change! 

httpx will continue to identify technology based on the [available data](https://github.com/projectdiscovery/wappalyzergo) before it becomes private.

In order to detect new technology, we need to switch to an alternative that consistently updates the latest tech data. Currently, we do not have any project in mind that can serve this purpose.
